bento/src/main/java/com/yelp/android/bento/core/SmartAsyncInflationCache.kt

```diff
@@ -0,0 +1,25 @@
+package com.yelp.android.bento.core
+
+/**
+ * Cache of how many of each view holder type's were actually inflated. This is a map of a name of
+ * a screen, to map of ViewHolder type's to how many view holder type's were requested.
+ */
+object SmartAsyncInflationCache : LinkedHashMap<String, MutableMap<Any, Int?>>() {
```

Comment:
Should we be worried about this cache taking up all the memory? Unless I missed something, we never clear it right? So we will track every viewholder ever in the app right?

Reply:
Yeah. I didn't think Class's and Int's would take up much memory, but I can add a cap of like 30 or 40?